Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    forever_style_guide (3.2.3)
```

Comment:
gemfile.lock is one of the two files we update to change the version. (for reference: https://github.com/forever-inc/forever-style-guide/pull/168/commits/54189dbaa38e92cf47bb90437afc2a6e35e373a0)